shared/src/iosMain/kotlin/co/touchlab/kampkit/NativeViewModel.kt

```diff
@@ -23,17 +24,19 @@ class NativeViewModel(
     private val log: Kermit by inject { parametersOf("BreedModel") }
     private val scope = MainScope(Dispatchers.Main, log)
     private val breedModel: BreedModel
+    private var currentJob: Job = Job()
```

Comment:
I'm still skeptical about `currentJob`, but if it is necessary and I'm missing something, NB this one isn't being used on update but the Android one is.

Reply:
Yeah, it's necessary, for the reasons expressed in the above comment.